### ISSUE TITLE
识别训练报错

### ISSUE THREAD

**Issue by opencv-wt** [Topdu/OpenOCR#110]
环境：四张V100服务器，unbuntu系统
使用官方finetune_rec.md文件中CUDA_VISIBLE_DEVICES=0,1,2,3 python -m torch.distributed.launch --nproc_per_node=4 tools/train_rec.py --c ./configs/rec/svtrv2/repsvtr_ch.yml命令。
报如下错误Expected to have finished reduction in the prior iteration before starting a new one. This error indicates that your module has parameters that were not used in producing loss. You can enable unused parameter detection by passing the keyword argument `find_unused_parameters=True` to `torch.nn.parallel.DistributedDataParallel`, and by making sure all `forward` function outputs participate in calculating loss.
请问作者有遇到相同的问题吗？怎么解决呀 @Topdu

**Comment by Topdu** [comment]
Try changing the value of `use_guide` in the configuration file to `False`:
```yaml
  Decoder:
    name: CTCDecoder
    svtr_encoder:
      dims: 256
      depth: 2
      hidden_dims: 256
      kernel_size: [1, 3]
      use_guide: False # True
```

**Comment by opencv-wt** [comment]
感谢作者，可以运行了
